tag_utils/cli/tag_delta.py

```diff
@@ -69,39 +85,41 @@ def main():
         print('component,' + lname + ',' + rname + ',status')
 
     # Downgrades (normally bad)
-    for c in sorted(delta_info['downgrades'].keys()):
-        lnevr = delta_info['downgrades'][c]['old']
-        rnevr = delta_info['downgrades'][c]['new']
-
-        levr = evr_from_nevr(lnevr)
-        revr = evr_from_nevr(rnevr)
-
-        if sys.stdout.isatty():
-            f = '%' + fw + 's %s%' + fw + 's %s%' + fw + 's'
-            print(f % (c, decor.HILIGHT, levr, decor.NORMAL, revr))
-        else:
-            if delta_info['downgrades'][c]['rebase']:
-                print(f'{c},{levr},{revr},rebase')
+    if show_downgrades:
+        for c in sorted(delta_info['downgrades'].keys()):
+            lnevr = delta_info['downgrades'][c]['old']
+            rnevr = delta_info['downgrades'][c]['new']
+
+            levr = evr_from_nevr(lnevr)
+            revr = evr_from_nevr(rnevr)
+
+            if sys.stdout.isatty():
+                f = '%' + fw + 's %s%' + fw + 's %s%' + fw + 's'
+                print(f % (c, decor.HILIGHT, levr, decor.NORMAL, revr))
             else:
-                print(f'{c},{levr},{revr},')
+                if delta_info['downgrades'][c]['rebase']:
+                    print(f'{c},{levr},{revr},rebase')
+                else:
+                    print(f'{c},{levr},{revr},')
 
-    for c in sorted(delta_info['upgrades'].keys()):
-        lnevr = delta_info['upgrades'][c]['old']
-        rnevr = delta_info['upgrades'][c]['new']
+    if show_upgrades:
+        for c in sorted(delta_info['upgrades'].keys()):
+            lnevr = delta_info['upgrades'][c]['old']
+            rnevr = delta_info['upgrades'][c]['new']
 
-        levr = evr_from_nevr(lnevr)
-        revr = evr_from_nevr(rnevr)
+            levr = evr_from_nevr(lnevr)
+            revr = evr_from_nevr(rnevr)
 
-        if sys.stdout.isatty():
-            f = '%' + fw + 's %' + fw + 's %s%' + fw + 's%s'
-            print(f % (c, levr, decor.HILIGHT, revr, decor.NORMAL))
-        else:
-            if delta_info['upgrades'][c]['rebase']:
-                print(f'{c},{levr},{revr},rebase')
+            if sys.stdout.isatty():
+                f = '%' + fw + 's %' + fw + 's %s%' + fw + 's%s'
+                print(f % (c, levr, decor.HILIGHT, revr, decor.NORMAL))
             else:
-                print(f'{c},{levr},{revr},')
+                if delta_info['upgrades'][c]['rebase']:
+                    print(f'{c},{levr},{revr},rebase')
+                else:
+                    print(f'{c},{levr},{revr},')
```

Comment:
Same as above:
The diff is having a hard time with this.  The block is indented one place right to be under the new `if show_upgrades:` conditional, there's no other changes.